index.js

```diff
@@ -22,6 +24,47 @@ require('./functions.js');
   server.listen(3000, () => console.log('website is alive'));
 }
 
+function $(process, ...args) {
+  const child = child_process.spawn(process, args);
+  return new Promise((resolve, reject) => {
+    let res = '';
+
+    child.stdout.on('data', (data) => {
+      res += data;
+    });
+
+    child.stderr.on('data', (data) => {
+      res += data;
+    });
+
+    child.on('close', (code) => {
+      if (code === 0) resolve(res);
+      else reject(res);
+    });
+  });
+}
+
+$.obj = function $obj(process, ...args) {
```

Comment:
should this perhaps use `Object.defineProperty`?

Reply:
sure why not